cms/static/js/programs/apps/program_admin_app.js

```diff
@@ -0,0 +1,15 @@
+(function() {
+    'use strict';
+
+    /**
+     * Allows for rapid toggling between app version locally
+     * i.e. just swap out the file name in the require array
+     */
```

Comment:
@rlucioni @schenedx I'm not convinced this should be added because the reasons for including it previously were to maintain a self-contained app, which by moving it into the studio code base we are decisively moving away from. I recommend just add core, what do you think?